src/workers/parser/getSymbols.js

```diff
@@ -123,6 +124,57 @@ function getFunctionParameterNames(path: SimplePath): string[] {
   return [];
 }
 
+function getFunctionSignature(
+  path: SimplePath | null,
+  signature: string | null
+) {
+  if (!path) {
+    return signature;
+  }
+
+  if (t.isFunctionDeclaration(path)) {
```

Comment:
If I'm reading this right, this will mean that the signature for a function inside of a function declaration can be the declaration itself? I might be missing an edge case. e.g.

````
function someThing(callback) {
  callback(() => {

  });
}
```
is `someThing` the expected signature for the arrow function?